common/add_host_in_memory_inventory.yml

```diff
@@ -38,8 +38,10 @@
     ansible_ssh_pipelining: "{{ add_host_in_memory_inventory_ssh_pipeline | default(false) }}"
     ansible_remote_tmp: "{{ add_host_in_memory_inventory_remote_tmp | default(omit) }}"
     ansible_shell_executable: "{{ add_host_in_memory_inventory_shell | default(omit) }}"
-
   register: add_host_result
+  no_log: "{{ no_need_log | default(false) }}"
```

Comment:
Can we use a more sensible var name like `hide_secrets`?

Reply:
Yes. It can be more specific. Maybe "not_gather_log" is better, since it will not display all the results not only secret.

Reply:
Got your point. Change it to "hide_secrets"